app/views/employees/avg_time.html.erb

```diff
@@ -10,7 +10,7 @@
         <%if employee.attendances.attendances_by_month_complete.exists? %>
           <div> <%= employee.name %></div>
           <div>
-            <%= employee.get_total_avg_time %>
+            <%= employee.get_total_avg_time.inspect %>
```

Comment:
What do you need to print here?

Reply:
Here, we expect that printed the average time that employee worked by month, but i forgot to remove the inspect, we are used it only to debugging.

I removed it now

DONE